### DISCUSSION TITLE
"Batch Count" equivalent?

### DISCUSSION THREAD

**Discussion by NoteToSelfFindGoodNickname** [mcmonkeyprojects/sd-infinity-grid-generator-script#103] [Q&A] [answered]
Hello!

When I use the Infinity Grid script, the "Batch Count" number box in automatic 11111 is ignored.
What would be the equivalent of "Batch Count" in Infinity Script?

Thank you!

**Comment by mcmonkey4eva** [comment, marked as answer]
Not valid to use within the context of grid generation. You can add an axis `Seed` as `1, 2, 3, 4` if you just want multiple variations of each image the way batching normally does
